packages/block-library/src/query-total/block.json

```
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
```

```suggestion
	"apiVersion": 3,
```

3 is the most up to date version.